resources/deployments/controller/controller_deployments.go

```diff
@@ -65,7 +65,7 @@ func (d *DeploymentsController) PostDeployment(w rest.ResponseWriter, r *rest.Re
 		return
 	}
 
-	d.view.RenderSuccessPost(w, r, id)
+	d.view.RenderSuccessPost(w, r, "./deployments/"+id)
```

Comment:
got it, yes we can do that, as long as all Location-providing endpoints conform to the same pattern. they do currently, and no breaking changes are in sight, so I'll make a fix.

(I went for a more flexible solution after working on devadm - mixes base urls e.g. `/foo/{id}` and `foo/{id}/status`, so depending on the EP, the Location pattern is different).